src/mono/wasm/Wasm.Build.Tests/Common/TestOutputWrapper.cs

```diff
@@ -18,7 +18,7 @@ public void WriteLine(string message)
         baseOutput.WriteLine(message);
         _outputBuffer.AppendLine(message);
         if (EnvironmentVariables.ShowBuildOutput)
-            Console.WriteLine(message);
+            Console.WriteLine(message);      
```

Comment:
nit: Remove extra spaces

Reply:
thanks, in a follow-up. I was in a hurry to start checking if this worked on the CI